src/main/java/com/sgordievskiy/spring/dto/TodoResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package com.sgordievskiy.spring.dto;
+
+import com.sgordievskiy.spring.model.Priority;
+import com.sgordievskiy.spring.model.Status;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
+@Data
+public class TodoResponseDto {
```

Comment:
Should we add no args constructor as well? Or maybe it worth to use record here?